### PR TITLE
We don't ship jdbc, or odbc

### PR DESCRIPTION
**Pull request by davecramer**
For building the installers this repo is not gone, just unlinked
from gpdb5

### CODE REVIEW

**Comment by cjcjameson** [comment]
Any doc changes needed to highlight their new location?

**Comment by davecramer** [comment]
Well we don't ship them so there is no "new" location. We ship clients from datadirect or they can get them from upstream

**Comment by hlinnaka** [comment]
There are still references to jdbc and odbc in gpAux/Makefile.

File: gpAux/Makefile [hunk 1]
```diff
@@ -667,19 +665,11 @@ else
 	# ---- load LD_LIBRARY_PATH with the GPDB library set and build the ODBC drivers ----
```
Comment (hlinnaka) [inline gpAux/Makefile]: comment still talks about ODBC

File: gpAux/Makefile [hunk 2]
```diff
@@ -667,19 +665,11 @@ else
...
 	cd $(INSTLOC)/ && \
 	  . ./greenplum_path.sh; \
-	  cd $(BLD_TOP)/client/odbc/src && \
-	    $(MAKE) all
 	# ---- copy GPDB fileset ----
 	mkdir -p $(CONNECTIVITYINSTLOC_LIB)
```
Comment (hlinnaka) [inline gpAux/Makefile]: What's left in the connectivity package after removing JDBC and ODBC? Libpq?

File: gpAux/Makefile [hunk 3]
```diff
@@ -667,19 +665,11 @@ else
...
 	(cd $(INSTLOC)/include && $(TAR) cf - *) | (cd $(CONNECTIVITYINSTLOC_INCLUDE)/ && $(TAR) xvpf -)$(check_pipe_for_errors)
-	# ---- copy JDBC fileset ----
-	mkdir -p $(CONNECTIVITYINSTLOC_JDBC)
-	cp -fp client/jdbc/*.jar $(CONNECTIVITYINSTLOC_JDBC)
-	# ---- copy ODBC fileset ----
-	mkdir -p $(CONNECTIVITYINSTLOC_ODBC)
-	(cd client/odbc/src/greenplum-connectivity/odbc/ && $(TAR) cf - --exclude=*.la *) | (cd $(CONNECTIVITYINSTLOC_ODBC)/ && $(TAR) xvf -)$(check_pipe_for_errors)
 	# ---- copy scripts fileset ----
 	mkdir -p $(CONNECTIVITYINSTLOC)
 	cp -f client/scripts/greenplum_$@_path$(SCRIPT) $(CONNECTIVITYINSTLOC)/
```
Comment (hlinnaka) [inline gpAux/Makefile]: The client/scripts directory is also from the submodule that's being removed.

Comment (hlinnaka) [inline gpAux/Makefile]: And so is the client/install stuff below. There's probably more, if you search for "client/" in this makefile.

Without the client submodule, is building the connectivity package completely kaput? Can we just remove it altogether?

Comment (davecramer) [inline gpAux/Makefile]: So yes, I think we can remove the client module completely

**Comment by davecramer** [comment]
@hlinnaka I've removed more of them, can you review?

File: gpAux/Makefile [hunk 4]
```diff
@@ -694,24 +681,18 @@ else
 	# ---- updating the version in required packages ----
 	@$(MAKE) set_scripts_version INSTLOC=$(CONNECTIVITYINSTLOC)
 
-ifneq "$(findstring win,$(BLD_ARCH))" "win"
```
Comment (hlinnaka) [inline gpAux/Makefile]: Removing this changes the logic. The else-branch used to be only for Windows, now we go there on all platforms.

File: gpAux/Makefile [hunk 5]
```diff
@@ -746,24 +720,18 @@ endif
 	# ---- Updating the version in required packages ----
 	@$(MAKE) set_scripts_version INSTLOC=$(CLIENTINSTLOC)
 
-ifneq "$(findstring win,$(BLD_ARCH))" "win"
```
Comment (hlinnaka) [inline gpAux/Makefile]: Same here, we now do the formerly windows-only actions on all platforms.

**Comment by danielgustafsson** [comment]
Can/should this PR also remove JDBC from BugBuster? See `src/test/regress/bugbuster/sql/jdbc.sql` and related schema/expected file.

File: gpAux/Makefile (hunk 2 above)
Comment (davecramer) [inline gpAux/Makefile]: I believe that is it

**Comment by davecramer** [comment]
I believe it should be removed from BugBuster as well

**Comment by Eulerizeit** [comment]
@davecramer what is left to be done before we merge this? Are you just waiting for @hlinnaka to review it?

**Comment by Eulerizeit** [comment]
@pivotal-mike or @hlinnaka could you review this please?

**Comment by danielgustafsson** [comment]
The jdbc test was removed by the #2109 BugBuster PR.

**Pull request merged**